packages/common-common/src/rabbitmq/scripts/publishRmqMessage.ts

```diff
@@ -0,0 +1,34 @@
+import { publishRmqMsg } from "common-common/src/rabbitmq/util";
+import {EventKind, ITransfer, IEventData} from "chain-events/src/chains/aave/types";
+import {CWEvent, SupportedNetwork} from "chain-events/src";
+import {v4 as uuidv4} from 'uuid';
+import {RascalExchanges, RascalRoutingKeys} from "common-common/src/rabbitmq";
+import {RABBITMQ_API_URI} from "commonwealth/server/config";
+
+async function publishRmqMessage() {
+  const ceData: ITransfer = {
+    kind: EventKind.Transfer,
+    tokenAddress: uuidv4(),
+    from: uuidv4(),
+    to: uuidv4(),
+    amount: uuidv4()
+  }
+  // // create a fake aave-transfer event
+  const chainEvent: CWEvent<IEventData> = {
+    blockNumber: Math.floor(Math.random() * 1000000),
+    data: ceData,
```

Comment:
how will `ceData` be passed in, in the real world situation? As an arg for `publishRmqMessage`?